xbmc/pvr/windows/GUIWindowPVRGuide.cpp

```diff
@@ -86,22 +86,30 @@ void CGUIWindowPVRGuide::GetContextButtons(int itemNumber, CContextButtons &butt
   buttons.Add(CONTEXT_BUTTON_INFO, 19047);              /* Programme information */
   buttons.Add(CONTEXT_BUTTON_FIND, 19003);              /* Find similar */
 
-  if (pItem->GetEPGInfoTag()->HasTimer())
+  CEpgInfoTagPtr epg(pItem->GetEPGInfoTag());
+  if (epg)
```

Comment:
Sorry to bother you again, but there is one more thing I just stumbled upon. Checking for <code>epg</code> in this method is not needed. Original code does not check using <code>HasEPGInfoTag</code> either. The epg tag has to be there in this context. Just Get it once and use it, please.